### DISCUSSION TITLE
Github SSO role mapping

### DISCUSSION THREAD

**Discussion by superbarne** [gravitational/teleport#14423] [Q&A] [answered]
Hi,

at the moment it is possible to map Github Teams to teleport roles. Is it also possible to define wich user is has wich role in teleport itself?

Maybe with role mapping template (teams_to_roles) in the github sso configuration: github-developer-{{external.logins}} ? This woul allow a role for each team member.

We want to grant specific users access to certain applications and not the whole team. Is this possible?

**Comment by webvictim** [comment, marked as answer]
As far as I know this isn't currently possible. You could achieve the same thing by creating a team per user in Github and then mapping that in the `teams_to_roles` configuration.
